### PR TITLE
Editorial changes for Chapter 1

### PR DESCRIPTION
**Pull request by g-murray**
Applied editorial suggestions for chapter 1 and 1.1 Editorial changes for chapter 1:

https://issues.redhat.com/browse/AAP-18141

Editorial edits: https://docs.google.com/document/d/1tRNUZxvBDFxPbiPDVGWEywvwV2Eta_5mhk-CvbQIObc/edit

### CODE REVIEW

File: downstream/assemblies/platform/assembly-platform-install-overview.adoc [hunk 1]
```diff
@@ -9,13 +9,13 @@ ifdef::context[:parent-context: {context}]
...
 
 Regardless of the installation scenario you choose, installing {PlatformNameShort} involves the following steps:
 
 xref:proc-editing-installer-inventory-file_platform-install-scenario[Editing the {PlatformName} installer inventory file]:: The {PlatformNameShort} installer inventory file allows you to specify your installation scenario and describe host deployments to Ansible. The examples provided in this document show the parameter specifications needed to install that scenario for your deployment.
 
-xref:proc-running-setup-script_platform-install-scenario[Running the {PlatformName} installer setup script]:: The setup script installs your Private Automation Hub using the required parameters defined in the inventory file.
+xref:proc-running-setup-script_platform-install-scenario[Running the {PlatformName} installer setup script]:: The setup script installs your private automation hub using the required parameters defined in the inventory file.
```
Comment (sayjadha) [inline downstream/assemblies/platform/assembly-platform-install-overview.adoc:18]: using -> by using

**Review by sayjadha — COMMENTED** [review]
One minor edit, rest LGTM.

File: downstream/assemblies/platform/assembly-platform-install-overview.adoc (hunk 1 above)
Comment (g-murray) [reply downstream/assemblies/platform/assembly-platform-install-overview.adoc]: Done, thank you!

**Comment by michellemacrh** [comment]
LGTM!

**Pull request merged**